css/styles.css

```
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+  font-family: "Courier New", Courier, monospace;
+}
+
+h1 {
+  text-transform: uppercase;
+  display: flex;
+  justify-content: center;
+}
+
+header {
+  background: #030007;
+  margin: 0px 0px 0px -8px;
+  color: #ffffff;
+  height: 60px;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+}
+
+h2 {
+  margin-top: 20px;
+}
+
+.question-card {
+  margin-top: 90px;
+  margin-bottom: 90px;
+  border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.div_question-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.question-card_categories {
+  display: flex;
+  list-style-type: none;
+  position: absolute bottom -5px left -5px;
```

```suggestion
  position: absolute; 
  bottom: -5px ;
  left: -5px;
```

there is no shorthand for this.